CODE_OF_CONDUCT.md

```diff
@@ -40,7 +40,7 @@ Project maintainers who do not follow or enforce the Code of Conduct in good fai
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4, available at [http://contributor-covenant.org/version/1/4][version]
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4, available at [https://www.contributor-covenant.org/version/1/4/code-of-conduct/][version]
```

Comment:
Why do we need to change this link? It does work, so I don't see how it's a typo.

Reply:
The original link is still usable, but redirect to this link.

Original: http://contributor-covenant.org/version/1/4  
HTTP 301 -> https://contributor-covenant.org/version/1/4  
HTTP 301 -> https://www.contributor-covenant.org/version/1/4  
HTTP 301 -> https://www.contributor-covenant.org/version/1/4/  
HTML meta refresh -> https://www.contributor-covenant.org/version/1/4/code-of-conduct/

I think it's better to use the new link instead of the old one.

Reply:
@marcusolsson Should we bother changing the CoC links?

Reply:
In order to get this PR through, I would remove this change since it's not a typo and it's unclear if it's a wise change or not.